docs/tiltfile_concepts.md

```diff
@@ -130,18 +130,15 @@ You can combine multiple optional arguments.
 ## Resources
 Tilt's UI makes it easier to find errors by grouping related status and output. E.g., when you edit a file, you want to know what error it caused, whether it's an error at build-time, deploy-time, or run-time. Tilt calls these groupings "Resources". Each Resource has a line in the UI that can be expanded and investigated.
 
-Tilt generates these groups after executing your `Tiltfile`. We're actively working on how to group in ways that make the most intuitive sense, so the specific algorithm is in-flux. We'll expand this paragraph when it's more settled.
+Tilt generates these groups after executing your `Tiltfile`. It does this by scanning all loaded yaml for any k8s objects that it considers a "workload" (i.e., it defines a Container, or is a CRD with an image\_json\_path). Each of these objects becomes a resource.
```

Comment:
Maybe worth making a separate (brief) section for the definition of "workload" so we can link from other docs?